packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+// 3rd party
+import {
+  Link as ReactAriaLink,
+  LinkProps as ReactAriaLinkProps,
+} from "react-aria-components";
+
+import { WalkthroughInfo } from "@repo/data/useWalkthroughData";
+import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps extends ReactAriaLinkProps {
+  id: string;
+  data: WalkthroughInfo;
+  walkthroughId: string;
```

Comment:
Can we add the data-testid prop like we have in the other components to match the paradigm? Lauren has said this is helpful for selection on her side for the cypress tests as well.